assets/js/bespoke.js

```diff
@@ -0,0 +1,9 @@
+window.onclick = function () {
```

Comment:
seems a bit hacky to trigger this on every button click on the page.

Reply:
we have total four buttons on this page.
1) Remove - trigger is needed
2) Add - trigger is needed
3) Continue and 4) Delete -> This will redirect any way to check answers page so there won't be much impact

Reply:
changed the implementation using overwrite method of existing class.